Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/core/mybatis-generator-systests-common/target/\r\n/core/mybatis-generator-maven-plugin/target/\r\n/core/mybatis-generator-systests-domtests/target/\r\n/core/mybatis-generator-systests-mybatis3/target/\r\n/core/mybatis-generator-core/target/\r\ndemo-mybatisplus/src/main/java/com/vgosoft/demomybatisplus/config/SwaggerGenerate.java\r\ndemo-mybatisplus/src/main/java/com/vgosoft/demomybatisplus/DemoMybatisplusApplication.java\r\ndemo-mybatisplus/src/main/resources/application.yml\r\ndemo-mybatisplus/src/main/resources/generatorConfig.xml\r\ndemo-mybatisplus/src/test/java/com/vgosoft/demomybatisplus/DemoMybatisplusApplicationTests.java\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore
+++ b/.gitignore
@@ -8,3 +8,14 @@
 demo-mybatisplus/src/main/resources/application.yml
 demo-mybatisplus/src/main/resources/generatorConfig.xml
 demo-mybatisplus/src/test/java/com/vgosoft/demomybatisplus/DemoMybatisplusApplicationTests.java
+.idea/compiler.xml
+.idea/jarRepositories.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/workspace.xml
+*.iml
+.idea/compiler.xml
+.idea/jarRepositories.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/workspace.xml
